http/src/main/resources/reference.conf

```diff
@@ -91,6 +91,7 @@ mysql {
     connectionTimeout = 5000
     numThreads = 50
   }
+  concurrency = 120
```

Comment:
Trying to understand how `concurrency` relates to `numThreads`. As I understand it slick uses a thread pool internally. So do we have this situation?
```
global ec --> (cached thread pool with 120-limit semaphore) --> (slick thread pool of 50)
```
I am wondering if `concurrency` and `numThreads` should be the same value…

Reply:
I think `numThreads` is probably physical threads, which is different from `IO`s. See https://typelevel.org/cats-effect/concurrency/basics.html#green-threads...so I think we can have more `IO`s than jvm threads.